openassessment/xblock/defaults.py

```diff
@@ -63,13 +63,13 @@
 # The rubric's feedback prompt is a set of instructions letting the student
 # know they can provide additional free form feedback in their assessment.
 DEFAULT_RUBRIC_FEEDBACK_PROMPT = """
-(Optional) What aspects of this response stood out to you? What did it do well? How could it improve?
+(Optional) What aspects of this response stood out to you? What did it do well? How could it be improved?
```

Comment:
@efischer19 Thanks! The screenshot showing the old text also shows an odd indent for this line. Does that issue still exist and if yes does this PR fix it?